docs/fastdds/use_cases/statistics_from_dedicated_interface/statistics_from_dedicated_interface.rst

```diff
@@ -0,0 +1,58 @@
+.. include:: ../../../03-exports/aliases-api.include
+
+.. _use-case-statistics-from-dedicated-interface:
+
+Receive Statistics data from a dedicated interface
+==================================================
+
+*eProsima Fast DDS* is designed to flexibly manage data transmission across different network configurations.
+The default behavior is attempting to transmit data to any remote locator for which a compatible transport
+is registered. This behavior facilitates robust connectivity across diverse network environments, but may
+not always align with specific user requirements. In some scenarios, the user may need to send specific data
```

Comment:
`may not always align` or `may not always be aligned`? Which is more common in english?

Reply:
I think that both of them can be used and are correct. "may not always be aligned" is more passive and could imply that the alignment is influenced by external factors, instead we want to specify that is a User decision. I would leave "may not always align", but since both are correct I am not against changing it.

Reply:
I think `may not always align` makes more sense in this case